### ISSUE TITLE
bug(@vtmn/css-checkbox): label display

### ISSUE THREAD

**Issue by Margot-ElMadi** [Decathlon/vitamin-web#643]
**Bug report**

<img width="428" alt="Capture d’écran 2021-10-01 à 16 05 04" src="https://user-images.githubusercontent.com/72196739/135637271-9f2f73d0-5934-42ac-b985-bbcdd029a2a3.png">

<img width="542" alt="Capture d’écran 2021-10-01 à 16 05 11" src="https://user-images.githubusercontent.com/72196739/135637466-dbf7a6af-c16b-4940-ba7d-622e2315bf36.png">

On mobile, Checkbox style is broken if we have html in our label.

I have fix the bug, but I can't test my job
https://github.com/Decathlon/vitamin-web/pull/644

**Comment by lauthieb** [comment]
Hi @Margot-ElMadi, I've found a solution for your problem. You just need to isolate your text inside `<span>` HTML tag, because we are using a div `display: inline-flex` in order to keep the whitespace to the left even if we are multiline.

Here is the solution for your use case:
```html
<input
      class="vtmn-checkbox vtmn-checkbox--indeterminate"
      type="checkbox"
      id="choice6"
      checked="checked"
      disabled="disabled"
    />
    <label for="choice6">Aikido</label>
  </div>
```

We've just updated our Storybook in favor to add this example. Thanks again for your contribution!
